en/peripherals/uavcan_escs.md

```diff
@@ -1,185 +1 @@
-   :::
-
-1. After the last motor is enumerated, the confirmation sound should change to indicate that the enumeration procedure is complete.
-
-The following video demonstrates the process:
-
-@[youtube](https://www.youtube.com/watch?v=4nSa8tvpbgQ)
-@[youtube](https://www.youtube.com/watch?v=4nSa8tvpbgQ)
-
-#### Manual ESC Enumeration using Sapog
-
-:::tip
-We recommend automated [ESC Enumeration using QGroundControl](#sapog_esc_qgc) rather than manual enumeration - it is easier and safer.
-:::
-
-You can manually configure the ESC index and direction using the [UAVCAN GUI Tool](https://uavcan.org/GUI_Tool/Overview/).
-This assigns the following Sapog configuration parameters for each enumerated ESC:
-- `esc_index`
-- `ctl_dir`
-
-:::note
-See [Sapog reference manual](https://files.zubax.com/products/io.px4.sapog/Sapog_v2_Reference_Manual.pdf) for more information about the parameters.
-:::
-
-### Myxa ESC Setup
-
-Motor enumeration for Myxa [Telega-based ESCs](https://zubax.com/products/telega) is usually performed using the [Kucher tool](https://files.zubax.com/products/com.zubax.kucher/) (or less "GUI-friendly" [UAVCAN GUI Tool](https://uavcan.org/GUI_Tool/Overview/)).
-
-There is some guidance here: [Quick start guide for Myxa v0.1](https://forum.zubax.com/t/quick-start-guide-for-myxa-v0-1/911) (Zubax blog).
-
-
-### VESC ESC Setup
-
-For [VESC ESCs](https://vesc-project.com/) the preferred tool for motor enumeration is the [VESC tool](https://vesc-project.com/vesc_tool).
-In addition to the normal motor configuration that you will have to setup in the VESC tool, you will also need to properly setup the app configuration.
-The recommended app setup is as follows:
-
-Parameter | Option
---- | ---
-App to use | `No App`
-VESC ID | `1,2,...`
-Can Status Message Mode | `CAN_STATUS_1_2_3_4_5`
-CAN Baud Rate | `CAN_BAUD_500K`
-CAN Mode | `UAVCAN`
-UAVCAN ESC Index | `0,1,...`
-
-
-VESC ID should have the same motor numbering as in PX4 convention, starting at `1` for top-right motor, `2` for bottom-left motor etc.
-However the `UAVCAN ESC Index` starts from `0`, and as such it is always one index lower than the `VESC ID`.
-For example, in a quadcopter the bottom left motor will have `VESC ID = 2` and `UAVCAN ESC Index = 1`.
-
-Finally the `CAN Baud Rate` must match the value set in [UAVCAN_BITRATE](../advanced_config/parameter_reference.md#UAVCAN_BITRATE).
-
-## Further Information
-
-- [PX4/Sapog](https://github.com/PX4/sapog#px4-sapog) (Github)
-- [Sapog v2 Reference Manual](https://files.zubax.com/products/io.px4.sapog/Sapog_v2_Reference_Manual.pdf)
-- [UAVCAN Device Interconnection](https://kb.zubax.com/display/MAINKB/UAVCAN+device+interconnection) (Zubax KB)
-- [Using Sapog based ESC with PX4](https://kb.zubax.com/display/MAINKB/Using+Sapog-based+ESC+with+PX4) (Zubax KB)
-
+<Redirect to="../uavcan/escs" />
```

Comment:
FYI only. I moved this doc under "/uavcan/. Wherever docs appear in the tree, I think it makes sense to group them under uavcan in the folder structure.